WalletWasabi.Fluent/Views/MainWindow.axaml.cs

```diff
@@ -11,6 +13,23 @@ public class MainWindow : Window
 	public MainWindow()
 	{
 		InitializeComponent();
+
+		// TODO: This is a workaround for this Avalonia issue (https://github.com/AvaloniaUI/Avalonia/issues/11850)
+		// Please, remove after it's been fixed.
+		ApplyRendererWorkaround();
```

Comment:
Make it a behavior please, easier to remove later and we should avoid touching code behind files.